sdk/cosmos/azure-cosmos/azure/cosmos/_location_cache.py

```diff
@@ -134,19 +126,19 @@ def __init__(
         refresh_time_interval_in_ms,
     ):
         self.preferred_locations = preferred_locations
-        self.default_regional_endpoint = RegionalEndpoint(default_endpoint, default_endpoint)
+        self.default_dual_endpoint = DualEndpoint(default_endpoint, default_endpoint)
         self.enable_endpoint_discovery = enable_endpoint_discovery
         self.use_multiple_write_locations = use_multiple_write_locations
         self.enable_multiple_writable_locations = False
-        self.write_regional_endpoints = [self.default_regional_endpoint]
-        self.read_regional_endpoints = [self.default_regional_endpoint]
+        self.write_dual_endpoints = [self.default_dual_endpoint]
+        self.read_dual_endpoints = [self.default_dual_endpoint]
         self.location_unavailability_info_by_endpoint = {}
         self.refresh_time_interval_in_ms = refresh_time_interval_in_ms
         self.last_cache_update_time_stamp = 0
-        self.available_read_regional_endpoints_by_location = {} # pylint: disable=name-too-long
-        self.available_write_regional_endpoints_by_location = {} # pylint: disable=name-too-long
-        self.available_write_locations = []
-        self.available_read_locations = []
+        self.account_read_dual_endpoints_by_location = {}
+        self.account_write_dual_endpoints_by_location = {}
+        self.account_write_locations = []
+        self.account_read_locations = []
```

Comment:
huge fan of the change from `available_write_locations` to just `account_write_locations` - makes much more sense